server/apps/main/urls.py

```diff
@@ -30,4 +30,5 @@
     path('make_research/<oh_file_id>/<file_uuid>/',
          views.make_research,
          name='make_research'),
+    path('my_stories/', views.my_stories, name="my-stories"),
```

Comment:
You'll also have to change `my-stories` to `my_stories`, otherwise django will throw a `NoReverseMatch` error.

Reply:
All noted, @jhlink! Thank you, I'll get them fix now.

Reply:
Hey @jhlink! I'm done working on the "My Stories Page", could you please review and let me know what else needs attention?